xbmc/guilib/FFmpegImage.cpp

```diff
@@ -225,7 +225,7 @@ bool CFFmpegImage::Decode(unsigned char * const pixels, unsigned int width, unsi
   if (m_width == 0 || m_height == 0 || format != XB_FMT_A8R8G8B8)
     return false;
 
-  if (!m_pFrame || !m_pFrame->data)
+  if (!m_pFrame || !m_pFrame->data[0])
```

Comment:
you can't access data[0] if you are not sure if data != nullptr. data is a pointer to an array of pointers